QEfficient/cloud/compile.py

```diff
@@ -61,8 +62,15 @@ def main(
     create_and_dump_specializations(
         batch_size=batch_size, prompt_len=prompt_len, ctx_len=ctx_len, path=specialization_json_path
     )
-    custom_io_file_path = os.path.join(os.path.dirname(onnx_path), "custom_io.yaml")
-    
+
+    # Select the customIO config based on the mx flag.
+    if mxint8:
+        custom_io_file_name = "custom_io_fp16.yaml"
+    else:
```

Comment:
names don't match the control flow.
if mxint8, filename is _fp16

Reply:
Done